### DISCUSSION TITLE
Flowbite for tailwind v1.9.6

### DISCUSSION THREAD

**Discussion by eusahn** [themesberg/flowbite#79] [Q&A]
Using Tailwind v1.9.6 as my app needs to keep IE 11 support.
Possible to install Flowbite?

Had errors finding tailwindcss/colors module.

**Comment by zoltanszogyenyi** [comment]
Hey @byshine,

We started developing Flowbite when Tailwind CSS was already in v2.x+.

I'm not sure at this point what are the compatibility specs for Tailwind CSS v1.x.
